docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+services:
+  mysql:
+    image: mysql:latest
+    container_name: mysql_container
+    environment:
+      MYSQL_ROOT_PASSWORD: 1234
+      MYSQL_DATABASE: app_db
+      MYSQL_USER: app_user
+      MYSQL_PASSWORD: 1234
+    volumes:
+      - mysql_data:/var/lib/mysql
+    ports:
+      - "3306:3306"
+
+  todolist:
+    build:
+      context: .
+    container_name: todolist_container
+    depends_on:
+      - mysql
+    ports:
+      - "8000:8000"
+    environment:
+      DATABASE_HOST: mysql
+      DATABASE_NAME: app_db
+      DATABASE_USER: app_user
+      DATABASE_PASSWORD: 1234
```

Comment:
The `DATABASE_PASSWORD` is also hardcoded as '1234'. For security reasons, consider using environment variables or Docker secrets to manage this sensitive information.